charts/kong/templates/_helpers.tpl

```diff
@@ -109,7 +109,7 @@ Create a single listen (IP+port+parameter combo)
 {{- define "kong.singleListen" -}}
   {{- $listen := list -}}
   {{- $listen = append $listen (printf "0.0.0.0:%d" (int64 .containerPort)) -}}
-  {{- range $param := .parameters | uniq }}
+  {{- range $param := .parameters | default (list) | uniq }}
```

Comment:
What is the reason for this change?
Does this error out if parameters is undefined?

Reply:
Yeah, see the last commit/CI log for the commit prior. Range over null generates an error.